_config.yml

```diff
@@ -20,6 +20,9 @@ analytics:
     agency: GSA    # Change this to your agency
     subagency: TTS,18F # Change this to your agency
 
+# enable search bar
+search_site_handle: 18F-site
```

Comment:
I had to add this to re-enable the search bar - not sure how confidential it is.